### ISSUE TITLE
Any plans for code releasing

### ISSUE THREAD

**Issue by ChongjianGE** [HFAiLab/FourCastNet#1]
Hi,

Thanks for the great implementation job on FourCastNet.
I wonder if there are any time plans for releasing the data and code.

Best,
G

**Comment by VachelHU** [comment]
We have released the code.

The data will be released soon.

**Comment by ChongjianGE** [comment]
Thanks for the reply!

**Comment by VachelHU** [comment]
16h

**Comment by ChongjianGE** [comment]
Hi @VachelHU,

I have noticed that the data has been released in the hfai.datasets.ERA5 toolkit. 
However, it seems that it's unavailable to get access to the hfai package through the following command.
`_pip3 install hfai --extra-index-url https://pypi.hfai.high-flyer.cn/simple --trusted-host pypi.hfai.high-flyer.cn_
`
I wonder if I need to get the official permission to access the hfai package.

Thanks.
